Google.Api.Generator.Tests/ProtoTests/GrpcServiceConfig/gapic_metadata.json

```diff
@@ -0,0 +1,63 @@
+{
+  "schema": "1.0",
+  "comment": "This file maps proto services/RPCs to the corresponding library clients/methods",
+  "language": "csharp",
+  "protoPackage": "testing.grpcserviceconfig",
+  "libraryPackage": "Testing.GrpcServiceConfig",
+  "services": {
+    "GrpcServiceConfigNoRetry": {
+      "clients": {
+        "grpc": {
+          "libraryClient": "GrpcServiceConfigNoRetryClientImpl",

```

Comment:
+1 to this being `GrpcServiceConfigNoRetryClient`

Reply:
👍